src/pages/steps/RichAlert.tsx

```diff
@@ -0,0 +1,29 @@
+import { Alert } from '@/ui/Alert';
+import { Box, parseSpacing } from '@/ui/Box';
+
+function RichAlert({ children, ...props }) {
```

Comment:
Is it necessary to create this new component? 
Is it not possible to use the regular `Alert` component?

Reply:
It's possible since it's what it does underneath but I noticed quite a few places had these HTML alerts and repeated the same manual CSS to style them since by default ul/li/etc don't appear styled in alerts so it looked like an ok mid-level component to add to me. I can perhaps rephrase it as some kind of variant? Or should I perhaps add the custom `css` to all `Alert`s?

Reply:
Yes I would prefer this approach.
What do you think @simon-debruijn

Reply:
I Agree embedding it in the `Alert` component would be the preferred approach. Maybe by adding using a `richText` prop